custom_components/hacs/__init__.py

```diff
@@ -127,19 +127,18 @@ async def async_startup():
         """HACS startup tasks."""
         hacs.enable_hacs()
 
-        for location in (
-            hass.config.path("custom_components/custom_updater.py"),
-            hass.config.path("custom_components/custom_updater/__init__.py"),
-        ):
-            if os.path.exists(location):
-                hacs.log.critical(
-                    "This cannot be used with custom_updater. "
-                    "To use this you need to remove custom_updater form %s",
-                    location,
-                )
-
-                hacs.disable_hacs(HacsDisabledReason.CONSTRAINS)
-                return False
+        try:
+            import custom_components.custom_updater
+        except ImportError:
+            pass
+        else:
+            hacs.log.critical(
+                "This cannot be used with custom_updater. "
```

Comment:
Is it clear what `This` means? Maybe say `HACS` instead?

Reply:
From what I remember, there has not (yet) been any confusion about it, but I applied your suggestion.